### PR TITLE
[CP]  Partial rendering of the show view for citizen project 

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FIG=docker-compose
-RUN=$(FIG) run --rm app
-EXEC=$(FIG) exec app
+RUN?=$(FIG) run --rm app
+EXEC?=$(FIG) exec app
```
Comment (thomasbisignani) [inline Makefile]: Can you fix this in another PR ?

Comment (B-Galati) [reply Makefile]: Sure -> see https://github.com/EnMarche/en-marche.fr/pull/1833

File: src/Controller/EnMarche/CitizenProjectController.php [hunk 2]
```diff
@@ -26,7 +26,13 @@ public function showAction(CitizenProject $citizenProject): Response
     {
         $this->disableInProduction();
 
-        return new Response();
+        $citizenProjectManager = $this->get('app.citizen_project.manager');
+
+        return $this->render('citizen_project/show.html.twig', [
+            'citizen_project' => $citizenProject,
+            'citizen_project_administrators' => $citizenProjectManager->getCitizenProjectAdministrators($citizenProject),
```
[the remark this answers is not in the document]
Comment (B-Galati) [reply src/Controller/EnMarche/CitizenProjectController.php]: Agree !

[the remark this answers is not in the document]
Comment (B-Galati) [reply src/Controller/EnMarche/CitizenProjectController.php]: Will do it later

**Pull request merged**